run-sidecar

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+CURSORLESS_FORK_REPO="git@github.com:cursorless-everywhere/cursorless.git"
+CURSORLESS_FORK_REV="main"
+CURSORLESS_FORK_NAME="pokey-phillco.cursorless-fork"
+
+CURSORLESS_SIDECAR_REPO="git@github.com:cursorless-everywhere/sidecar.git"
+CURSORLESS_SIDECAR_REV="main"
+CURSORLESS_SIDECAR_NAME="phillco.cursorless-sidecar"
+
+if ! [ -x "$(command -v code)" ]; then
+    echo 'Error: code is not installed.' >&2
```

Comment:
```suggestion
    echo 'ERROR: VSCode cli is not installed. Please install VSCode https://code.visualstudio.com/download and ensure `code` CLI is available https://code.visualstudio.com/docs/editor/command-line#_launching-from-command-line' >&2
```

Alternately, we might consider doing the same thing we do in CI to automatically download and install VSCode and cli. See https://www.npmjs.com/package/@vscode/test-electron. Maybe let's leave that for follow-up work though